client/app/dbaas/logs/detail/streams/add/logs-streams-add.html

```diff
@@ -1,4 +1,148 @@
-<div>
-    Add Stream
-    Service Name: {{ ctrl.serviceName }}
+<div class="cui-page__content">
+    <oui-back-button data-state="dbaas.logs.detail.streams"
```

Comment:
You should use NavigationService.previousState or $window.history.back()

Reply:
Back button must always take user back to streams home page. So I can not use NavigationService.previousState.go() or $window.history.back().

When user visits edit stream page from bookmark url, using $window.history.back() will break

Reply:
Answer in previous comment